Cue/app/tabs/library/LibraryListView.js

```diff
@@ -110,7 +116,7 @@ class LibraryListView extends React.Component {
       return (
         <EmptyView
           icon={CueIcons.emptyLibrary}
-          titleText={'You don&rsquo;t have any decks yet.'}
+          titleText={'You don\'t have any decks yet.'}
```

Comment:
Actually in PR #93 this was changed to a proper curly quote, so you don't have to worry about this.